src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -117,6 +117,11 @@
      */
     private int firestoreQueryBatchSize;
 
+    /**
+     * Maximum number of DRS lookup requests allowed
+     */
+    private int maxDRSLookups;
```

Comment:
Not sure what the convention is elsewhere in the jade code base but normally all caps acronyms like `DRS` are camel cased as `Drs`